packages/suite/src/views/settings/general/SettingsGeneral.tsx

```diff
@@ -38,6 +40,10 @@ export const SettingsGeneral = () => {
             enabledNetworks.includes(symbol) && features?.includes('amount-unit'),
     );
 
+    const isMetadataEnabled = metadata.enabled && !metadata.initiating;
+    const isProviderConnected =
+        metadata.enabled && device?.metadata.status === 'enabled' && !!metadata.provider;
+
```

Comment:
maybe a selector similar to`selectIsLabelingAvailable` ?

Reply:
Not sure about it, since it's the only place this condition is used and it wouldn't save any lines really